.github/ISSUE_TEMPLATE/Bug.yml

```
@@ -0,0 +1,31 @@
+name: Bug
+description: Report an issue that would help to improve this project.
+labels: [' goal: fix']
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add any screenshots if applicable
```

For a chess engine, I guess the logs are more informative than screenshots, may be worth asking for that instead.

```suggestion
    attributes:
      label: Logs
      description: Please add the logs with `RUST_BACKTRACE=1` and `RUST_LOG=info`.
```